README.md

```diff
@@ -1,2 +1,70 @@
-# project
-Zambezi project template
+Zambezi Project
+===============
+
+Whenever you're starting a project, there are about a million questions that need to be answered – some of them inevitable end up having to be answered for every project. This template attempts to answer the most common ones, providing a solid foundation for you to build on.
+
+Usage
+-----
+
+Do *not* fork this project, unless you intend to actually [contribute changes](CONTRIBUTING.md) which is more than welcome!
+
+To create a new project:
+
+1. Clone (or download) this project: `git clone git@github.com:zambezi/project.git your-project`
```

Comment:
You're right, and when you're right, you're right! :o)

Updated to *much* simpler instructions in d7ac692.